### PR TITLE
style(api-client): more guided empty state in app

### PR DESCRIPTION
**Pull request by cameronrohani**
Add 'import collection' to empty state
image

+ big command k ascii art if <= 1 request

image

### CODE REVIEW

**Review by amritk — APPROVED** [review]
Seems legit, tested and working 👍🏾

File: packages/api-client/src/views/Request/ResponseSection/ResponseEmpty.vue [hunk 1]
```diff
@@ -57,7 +94,7 @@ onBeforeUnmount(() => events.hotKeys.off(handleHotKey))
         <ScalarHotkey hotkey="↵" />
       </button>
       <button
-        v-if="!isReadOnly && layout === 'desktop'"
+        v-if="layout !== 'modal'"
```
Comment (antlio) [inline packages/api-client/src/views/Request/ResponseSection/ResponseEmpty.vue:97]: should we keep the desktop condition here? so we do not display the meta+n on web?

Comment (cameronrohani) [reply packages/api-client/src/views/Request/ResponseSection/ResponseEmpty.vue:97]: feel free to adjust if you think it makes sense to do so

Comment (antlio) [reply packages/api-client/src/views/Request/ResponseSection/ResponseEmpty.vue:97]: ok will do! cause we can't use meta+n in web

**Pull request merged**